scala-package/core/src/test/scala/org/apache/mxnet/IOSuite.scala

```diff
@@ -99,7 +101,9 @@ class IOSuite extends FunSuite with BeforeAndAfterAll {
       "data_shape" -> "(3,28,28)",
       "batch_size" -> "100",
       "preprocess_threads" -> "4",
-      "prefetch_buffer" -> "1"
+      "prefetch_buffer" -> "1",
```

Comment:
create new tests that take new parameters, so we know that the existing APIs don't break.

Reply:
?